### PR TITLE
feat(qc-iot): update IOT sensors tab and IOT graph

### PR DESCRIPTION
**Pull request by kennethliporada**
# Description Overall
- IOT Sensors Tab
> - Change options to checkbox button
> - Set Flood Sensor as default layer once IOT Sensor is clicked
> - Change naming for options for IOT
> - Finalize legend color for dots

- IOT Graph/Chart

- > Add burger menu for download image and csv
- > Change background color of data in graph
- > Change datetime display in Y-axis
- > X-axis should be the value of the data
- > Add title as location inside the graph
- > Can hide/show when you click the flood sensor inside chart pop-up
- > Once the user hover on the dots, it shows the pop-up summary details
- > On click of the dots, it shows pop-up graph/chart
- > Once the graph/chart pop-up, the user can hover inside the graph/chart
- > Change size of the dots in the map as per instructions by Sir mahar

### CODE REVIEW

**Comment by kennethliporada** [comment]
# Description
- fix css for X button upper right side of the chart

# Screenshot
- Old
image

- New
- 
image

**Comment by kennethliporada** [comment]
# Description
- close button inserted margin right

# Photo
image

# Additional edited
- I also put a code on the sensor because when I put a code for the sensor font, so what I put on the iot is what I put into the sensor.

**Comment by kennethliporada** [comment]
# Description
- changed base URL to 'https://noah-api.up.edu.ph/'

image

**Review by bon-carpo — CHANGES_REQUESTED** [review]
delete line 64 and 66

File: src/app/features/noah-playground/services/qc-sensor-chart.service.ts [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+      },
+      series: [
+        {
+          name: 'Temperature',
+          color: '#0C2D48',
+          data: [],
+        },
+      ],
+    };
+  }
```
Comment (bon-carpo) [inline src/app/features/noah-playground/services/qc-sensor-chart.service.ts:326]: remove all  unused line/s

File: src/app/features/noah-playground/services/qc-sensor-chart.service.ts [hunk 2]
```diff
@@ -0,0 +1,573 @@
...
+      },
+      series: [
+        {
+          name: 'Sensor 1',
+          color: '#0C2D48',
+          data: [],
+        },
+      ],
+    };
+  }
```
Comment (bon-carpo) [inline src/app/features/noah-playground/services/qc-sensor-chart.service.ts:571]: @kennethliporada remove all unused line/s

File: src/app/features/noah-playground/components/qc-sensors-group/qc-sensors-group.component.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      *ngFor="let qcSensorType of qcWeatherTypes"
+      [qcSensorType]="qcSensorType"
+    ></noah-qc-sensor-solo>
+
+    <!-- <div class="pl-9">
+      <p class="text-xs italic mt-2">
+        Sensors data are from the stations deployed by DOST-ASTI, together with
+        the help of DOST and PAGASA
+      </p>
+    </div> -->
```
Comment (bon-carpo) [inline src/app/features/noah-playground/components/qc-sensors-group/qc-sensors-group.component.html:92]: remove line 87 to 92

File: src/app/features/noah-playground/components/qc-sensor-solo/qc-sensor-solo.component.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+import { Observable, Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+export const QC_SENSOR_NAMES: Record<QuezonCitySensorType, string> = {
+  humidity: 'Relative Humidity',
+  pressure: 'Air Pressure',
+  temperature: 'Temperature (Celsius)',
+  distance_m: 'Flood Height',
+  // distance: 'Distance',
+  // sensor_1: 'Sensor_1'
```
Comment (bon-carpo) [inline src/app/features/noah-playground/components/qc-sensor-solo/qc-sensor-solo.component.ts:13]: remove line 12 and 13

File: src/app/features/noah-playground/components/qc-sensor-solo/qc-sensor-solo.component.scss [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+}
+// .sensor_1 {
+//   --color: #F0BB62;
+//   background-color: #F0BB62;
+//   &,
+//   input {
+//     border-radius: 100% !important;
+//     --border-color: #9ca3af;
+//   }
+// }
```
Comment (bon-carpo) [inline src/app/features/noah-playground/components/qc-sensor-solo/qc-sensor-solo.component.scss:69]: remove all unused line/s

File: src/app/features/noah-playground/components/qc-sensor-solo/qc-sensor-solo.component.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        type="checkbox"
+        class="h-5 w-5 text-blue-600"
+        name="input-{{ qcSensorType }}"
+      />
+    </div>
+    <label class="ml-3" for="input-{{ qcSensorType }}">
+      {{ qcSensorName }}
+    </label>
+  </div>
+</div> -->
```
Comment (bon-carpo) [inline src/app/features/noah-playground/components/qc-sensor-solo/qc-sensor-solo.component.html:46]: remove line 30 to 46

File: src/app/features/noah-playground/components/map-playground/map-playground.component.ts [hunk 7]
```diff
@@ -89,11 +105,15 @@ type LH2Subtype = 'af' | 'df';
 
 // hazardOpacity$: Observable<number>;
 // hazardShown$: Observable<boolean>;
```
Comment (bon-carpo) [inline src/app/features/noah-playground/components/map-playground/map-playground.component.ts:107]: remove line 106 and 107

**Review by bon-carpo — CHANGES_REQUESTED** [review]
@kennethliporada kindly check my comments. Thank you

**Comment by kennethliporada** [comment]
# Description
- Deleted unused codes and comments

**Pull request merged**